content/terraform-enterprise/1.1.x/docs/enterprise/releases/1.0.x/index.mdx

```diff
@@ -133,7 +133,7 @@ Flexible Deployment Options `terraform-enterprise` container digest: amd64/linux
 
 ## Known Issues
 1. (Updated 11/27/2025) Product usage may generate heavy database queries that can overload the database and impact production workloads. This issue is fixed in 1.0.3 and in 1.1.0
-1. (Updated 09/18/2025) You may experience failures when using an S3-compatible storage solution. An AWS library we upgraded in this release is triggering authentication issues with some third-party storage solutions. As a result, you may experience errors when running plans and applies or when accessing Terraform state files. Refer to [issue 2960](https://github.com/aws/aws-sdk-go-v2/discussions/2960) on the AWS SDK GitHub issues page for more information.  
+1. (Updated 12/15/2025) Users may experience failures when using certain S3-compatible storage solutions. An AWS library upgrade in this release introduces authentication issues with some third-party storage providers. To prevent errors when running plans, applies, or accessing Terraform state files, upgrade your storage solution to a version compatible with AWS SDK for Go v2. For additional details, refer to [issue 2960](https://github.com/aws/aws-sdk-go-v2/discussions/2960) on the AWS SDK GitHub issues page.
```

Comment:
```suggestion
1. (Updated 12/15/2025) You may experience failures when using some S3-compatible storage solutions. This release includes an AWS library upgrade that introduces authentication issues with some third-party storage providers. To prevent errors when running plans, applies, or accessing Terraform state files, upgrade your storage solution to a version compatible with AWS SDK for Go v2. For additional details, refer to [issue 2960](https://github.com/aws/aws-sdk-go-v2/discussions/2960) on the AWS SDK GitHub issues page.
```